app/scripts/controllers/eventDetails.js

```diff
@@ -167,7 +167,7 @@ angular.module('confRegistrationWebApp')
       }
 
       //allowEditRegistrationAfterComplete
-      if ($scope.conference.allowEditRegistrationAfterComplete && !$scope.conference.requireLogin) {
+      if ($scope.conference.allowEditRegistrationAfterComplete && !($scope.conference.relayLogin || $scope.conference.facebookLogin)) {
```

Comment:
It may be a good idea to move this check `$scope.conference.relayLogin || $scope.conference.facebookLogin` into an attribute on conference.